vaadin-map-flow-parent/vaadin-map-flow/src/main/resources/META-INF/resources/frontend/vaadin-map/util.js

```diff
@@ -0,0 +1,17 @@
+import VectorSource from "ol/source/Vector";
+
+/**
+ * Searches an OpenLayers map instance for the layer whose source contains a specific feature
+ * @param map an OpenLayers map instance
+ * @param feature the feature that should be contained in the layers source
+ * @returns {*} the layer that contains the feature, or null
```

Comment:
`find` returns `undefined` when no element is satisfied with condition.

Reply:
Fixed the JSDoc